test/smoke-dev/occupancy-based-opt-xteam-reduction/Makefile

```diff
@@ -0,0 +1,19 @@
+include ../../Makefile.defs
+
+TESTNAME     = occupancy_based_opt_xteam_reduction
+TESTSRC_MAIN = occupancy_based_opt_xteam_reduction.c
+TESTSRC_AUX  =
+TESTSRC_ALL  = $(TESTSRC_MAIN) $(TESTSRC_AUX)
+RUNENV      += LIBOMPTARGET_KERNEL_TRACE=1
+RUNENV      += OMPX_XTEAMREDUCTION_OCCUPANCY_BASED_OPT=1
+CFLAGS      += -fopenmp-target-ignore-env-vars -fopenmp-assume-no-nested-parallelism -fopenmp-assume-no-thread-state
```

Comment:
I don't think these options are needed any more. Can you remove the CFLAGS and check whether you get the same result?

Reply:
Tested locally without the flag. It also works. Removed in the revision.